### PR TITLE
Update docs to reflect removal of PaginatorComponent.

### PR DESCRIPTION
**Pull request by ADmad**
Refs cakephp/cakephp#16124

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
cc @markstory

File: en/controllers/pagination.rst [hunk 1]
```diff
@@ -139,39 +124,16 @@ this count query can be very expensive. In situations where you only want to
...
-        ]);
+        protected array $paginate = [
+            'paginator' => \Cake\Datasource\SimplePaginator::class,
+        ];
     }
 
 When using the ``SimplePaginator`` you will not be able to generate page
 numbers, counter data, links to the last page, or total record count controls.
 
-Using the PaginatorComponent Directly
```
Comment (markstory) [inline en/controllers/pagination.rst:155]: Should we have a section on how to use Paginator directly?

**Comment by othercorey** [comment]
The deprecation was extended to 6.0, so this is probably not happening.

File: en/controllers/pagination.rst (hunk 1 above)
Comment (ADmad) [reply en/controllers/pagination.rst:155]: Sorry, I missed your comment and forgot about this PR. Will look into it soon.

**Comment by ADmad** [comment]
@othercorey We should still update the docs showing the new usage, the component removal is pushed back just to make upgrading easier.

**Comment by othercorey** [comment]
> @othercorey We should still update the docs showing the new usage, the component removal is pushed back just to make upgrading easier.

Definitely. I just didn't see it in this PR. Just poking so it got attention.

**Comment by ADmad** [comment]
I have added a section for using a paginator directly.

**Pull request merged**